FWCore/Services/src/SiteLocalConfigService.cc

```diff
@@ -329,8 +340,17 @@ namespace edm {
             if (frontierConnect) {
               m_frontierConnect = _toParenString(*frontierConnect);
             }
+            auto localConnect = calibData->FirstChildElement("local-connect");
+            if (localConnect) {
+              auto connectString = localConnect->FirstChildElement("connectString");
+              if (connectString) {
+                m_localConnectPrefix = safe(connectString->Attribute("prefix"));
+                m_localConnectSuffix = safe(connectString->Attribute("suffix"));
+              }
+            }
```

Comment:
I would make the `<frontier-connect>` and `<local-connect>` elements mutually exclusive.

Reply:
I will make the change such that the SiteLocalConfigService constructor will throw an exception if both <frontier-connect>  and <local-connect> are present. This seems reasonable to me.

This means someone creating a new XML file for one of these sites will have to both delete the <frontier-connect> section and add the <local-connect>. They will not be able to be lazy and just add <local-connect> and the other one then gets ignored. It is more work to edit the XML, but should be easier for someone to understand what is happening when reading the XML file.